app/controllers/unattended_controller.rb

```diff
@@ -171,7 +171,11 @@ def verify_found_host
     end
 
     error = host_verifier.errors.first
-    render_error(error[:type], error[:message], error[:params])
+    if ipxe_request?
```

Comment:
Doesn't `render_error` already handle this path correctly?

Reply:
Hmmm refactoring of refactored code and you end up with this ;-)